test/range.js

```diff
@@ -95,9 +97,14 @@ test('partial range', function (t) {
     request.get(opts, function (err, res, body) {
       t.ifError(err);
       t.equal(res.statusCode, 206, 'partial content status code');
-      t.equal(body, 'e!!</b>\n');
+      t.equal(eol.lf(body), 'e!!</b>\n');
```

Comment:
yep. On windows new line is \r\n or CRLF. On *nix it is \n or LF. I've also heard that ancient OS's uses \r OR CR. But I've never seen it IRL.